src/components/Popups/ReplyField/ReplyForm.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Button from 'box-ui-elements/es/components/button';
+import PrimaryButton from 'box-ui-elements/es/components/primary-button';
+import { EditorState } from 'draft-js';
+import { FormikProps } from 'formik';
+import isEmpty from 'lodash/isEmpty';
+import { FormattedMessage, useIntl } from 'react-intl';
+import ReplyField from './ReplyFieldContainer';
+import { FormValues, Props as OtherProps } from './ReplyFormContainer';
+import messages from '../messages';
+
+type Props = OtherProps & FormikProps<FormValues>;
+
+const ReplyForm = ({
+    errors,
+    handleSubmit,
+    isPending,
+    onCancel,
+    onChange,
+    setFieldValue,
+    values,
+}: Props): JSX.Element => {
+    const intl = useIntl();
+    const hasErrors = !isEmpty(errors);
+    const handleChange = (editorState: EditorState): void => {
+        const text = editorState.getCurrentContent().getPlainText();
+        onChange(text);
+        setFieldValue('editorState', editorState);
+    };
+
+    return (
+        <form className="ba-Popup-form" data-testid="ba-Popup-form" onSubmit={handleSubmit}>
+            <div className="ba-Popup-main">
+                <ReplyField
```

Comment:
We should use Formik's `Field` [component](https://jaredpalmer.com/formik/docs/api/field) to manage the `onChange` lifecycle for the primitive ReplyField component. We lose out on some of the value of the library without it.

Reply:
Agreed, but it due to the nature of getting DraftJS to work (setting the `editorState`), we need to use the `setFieldValue` helper instead of the default `onChange` that gets wired up with the `Field` component